.github/workflows/Build.yml

```diff
@@ -15,6 +15,14 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  get-pr-number:
+    if: github.event_name == 'pull_request'
+    name: "get-pr-number"
+    runs-on: ubuntu-latest
+    steps:
+      - name: ${{ github.event.pull_request.number }}
+        run: echo ok
```

Comment:
Is this debug code? 👀

Reply:
Nope, all we need here is to leave the pull request number for further reference in `Stage*.yml`. Although nothing needs to be run here, steps require doing something.